package.json

```diff
@@ -16,7 +16,8 @@
     "reinstall": "npm run clean && npm install",
     "test": "lerna exec -- npm run test",
     "tsc": "lerna exec -- npm run tsc",
-    "tsc.clean": "npx lerna exec -- npx shx rm -rf lib dist"
+    "tsc.clean": "npx lerna exec -- npx shx rm -rf lib dist",
+    "start": "lerna exec --scope @ganache/cli -- node --inspect lib/src/cli.js"
```

Comment:
```suggestion

```

Let's remove this, since it will be in conflict with #721.

Reply:
applied in another commit due to your suggestion leaving a trailing comma